service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -261,8 +257,7 @@ private ApiAzureLandingZoneResult toApiAzureLandingZoneResult(
         .landingZone(azureLandingZone);
   }
 
-  public UUID getLandingZoneId(BearerToken token, UUID workspaceId) {
-    Workspace workspace = workspaceService.getWorkspace(workspaceId);
+  public UUID getLandingZoneId(BearerToken token, Workspace workspace) {
     Optional<UUID> profileId = workspace.getSpendProfileId().map(sp -> UUID.fromString(sp.getId()));
```

Comment:
FYI - I am in process of moving spend profiles to the cloud context and out of the workspace. As per, [New Create/Delete Workspace Endpoints](https://docs.google.com/document/d/1zbNppE6JJWbFGbasO5iHZHg3EjFVqHeOuj8MULP11qo/edit?resourcekey=0-Qm_6ORckNLekwOSqQ6f83Q#heading=h.potz6upymld7).